website/docs/r/logs_alert.html.markdown

```diff
@@ -13,11 +13,18 @@ Create, update, and delete logs_alerts with this resource.
 
 ## Example Usage
 ```hcl
+
 resource "ibm_resource_instance" "logs_instance" {
   name     = "logs-instance"
   service  = "logs"
-  plan     = "experimental"
-  location = "eu-gb"
+  plan     = "standard"
+  location = "eu-de"
+  parameters = {
+    logs_bucket_crn         = ibm_cos_bucket.logs_bucket.crn // Eg: crn:v1:bluemix:public:cloud-object-storage:global:a/4448261269a14562b839e0a3019ed980:f8b3176e-af8e-4e14-a2f9-7f82634e7f0b:bucket:logs-bucket
+    logs_bucket_endpoint    = ibm_cos_bucket.logs_bucket.s3_endpoint_direct // Eg: s3.direct.eu-de.cloud-object-storage.appdomain.cloud
+    metrics_bucket_crn      = ibm_cos_bucket.metrics_bucket.crn
+    metrics_bucket_endpoint = ibm_cos_bucket.metrics_bucket.s3_endpoint_direct
```

Comment:
Do we need this example in `logs_alert` document?

Reply:
Not required. Just in case people miss seeing resource_instance doc
I can remove it though